packages/eventindexer/CHANGELOG.md

```diff
@@ -0,0 +1,63 @@
+# Changelog
```

Comment:
we can delete this file, release-please should generate a new one once this is merged in